conf/machine/olinuxino-a20lime2-emmc.conf

```diff
@@ -0,0 +1,12 @@
+#@TYPE: Machine
+#@NAME: Olimex A20-OLinuXino Lime2 Board with 4GB eMMC
+#@DESCRIPTION: Machine configuration for the Olimex A20-OLinuXino Lime2 Board with 4GB eMMC, based on Allwinner A20 CPU
+#https://github.com/OLIMEX/OLINUXINO
+
+require conf/machine/include/sun7i.inc
+
+PREFERRED_VERSION_linux = "4.11.0+git%"
```

Comment:
I elected to use 4.11 as 4.9 was not working for me on the Lime2 eMMC variant, even if it has the appropriate dts. I think it is due to the fact that it 4.9 is missing this fix with respect to 4.11:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/arch/arm/boot/dts/sun7i-a20-olinuxino-lime2-emmc.dts?h=v4.11-rc6&id=3116d37651d77125bf50f81f859b1278e02ccce6
If you want I can add a comment about it, to make it explicit why I used 4.11. Otherwise, if you really want to keep 4.9 I can backport the patch of the dts fix.